<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccepteCallGUI</class>
 <widget class="QWidget" name="AccepteCallGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AccepteCallGUI</string>
  </property>
  <widget class="QPushButton" name="reject_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">    background: #FE0000;
	border: solid 1px #236036;
    border-bottom: solid 3px #225f33;
    box-shadow: inset 0 0 0 1px #90a59c;
    color: #fff;
    text-shadow: 0 1px 0 #225f33;
	border-radius: 8px;</string>
   </property>
   <property name="text">
    <string>Reject</string>
   </property>
  </widget>
  <widget class="QPushButton" name="accpt_btn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">    background: #cae285;
    border: solid 1px #aad063;
    border-bottom: solid 3px #799545;
    box-shadow: inset 0 0 0 1px #e0eeb6;
    color: #5d7731;
    text-shadow: 0 1px 0 #d0e5a4;
	border-radius: 8px;</string>
   </property>
   <property name="text">
    <string>Accpept</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Appel entrant</string>
   </property>
  </widget>
  <widget class="QPushButton" name="end_call">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/LogGUI/end-call.png);
background-repeat: no-repeat;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LogGUI.qrc"/>
 </resources>
 <connections/>
</ui>
